modules/extra/deprecations.nix

```diff
@@ -59,5 +59,10 @@ in {
       With Trouble having so many different modes, and breaking changes
       upstream, it no longer makes sense, nor works, to toggle only Trouble.
     '')
+
```

Comment:
```suggestion
# 2024-11-30
```

It's good to note the date of deprecation in order to allow pinpointing the date of the commit.